builder/install/10-protobuf.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-curl -L "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-cpp-${PROTOBUF_VERSION}.tar.gz" | tar xz
+curl -L "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOBUF_VERSION}/protobuf-cpp-${PROTOBUF_VERSION}.tar.gz" | tar xz --no-same-owner
```

Comment:
Could you elaborate more (maybe in the comment, or the commit message) in which way is it related to the podman vs docker compatibility?

Reply:
Sure, this is related to running the container as non-root and the corresponding mapping to the actual user id.
If the [commit log](https://github.com/stackrox/collector/commit/f0facf5cc09077ac6efa628e2bea86a1c38e3764) should go further in the details, please tell me and I will elaborate.